server/ruby/setup.rb

```diff
@@ -0,0 +1,84 @@
+require 'stripe'
+require 'dotenv'
+
+Dotenv.load(File.dirname(__FILE__) + '/../../.env')
+
+Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+Stripe.api_version = '2019-02-11'
+
+def create_data
+  begin
+    products = [
+      {
+        id: 'increment',
+        type: 'good',
+        name: 'Increment Magazine',
+        attributes: ['issue']
+      },
+      {
+        id: 'pins',
+        type: 'good',
+        name: 'Stripe Pins',
+        attributes: ['set']
+      },
+      {
+        id: 'shirt',
+        type: 'good',
+        name: 'Stripe Shirt',
+        attributes: ['size', 'gender']
+      }
+    ]
+
+    products.each do |product|
+      Stripe::Product.create(product)
+    end
+
+    skus = [
+      {
+        id: 'increment-03',
+        product: 'increment',
+        attributes: {
+          issue: 'Issue #3 “Development”'
+        },
+        price: 399,
+        currency: 'usd',
```

Comment:
We should add a note to the README that the currency is hardcoded in the setup script.